client/my-sites/media-library/drop-zone.jsx

```diff
@@ -8,7 +8,7 @@ var React = require( 'react' ),
  * Internal dependencies
  */
 var analytics = require( 'lib/analytics' ),
-	DropZone = require( 'components/drop-zone' ),
+	DropZone = require( 'components/drop-zone' ).default,
```

Comment:
This section is small enough that it makes sense to tidy up the imports here instead. :)